debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/YugabyteDBConnector.java

```diff
@@ -61,10 +62,70 @@ public void start(Map<String, String> props) {
 
     @Override
     public List<Map<String, String>> taskConfigs(int maxTasks) {
+        if (props == null) {
+            return Collections.emptyList();
+        }
+
+        if (props.containsKey("snapshot.mode") && props.get("snapshot.mode").equalsIgnoreCase("parallel")) {
+            LOGGER.info("Initialising parallel snapshot consumption");
+
+            final String tableIncludeList = props.get("table.include.list");
```

Comment:
Instead of using strings for `snapshot.mode`, `table.include.list` and `parallel`, use the static values from PostgresConnectorConfig. For ex: `PostgresConectorConfig.TABLE_INCLUDE_LIST`